src/pages/components/toggle/style.mdx

```diff
@@ -26,10 +43,13 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
 </Column>
 </Row>
 
-<Caption>On and off toggle states</Caption>
+<Caption>On and off toggle states.</Caption>
```

Comment:
Hi @alina-jacob, should we be using a full stop/period at the end of this caption, considering it's not a complete sentence?

Reply:
I think, following what's mentioned in [this thread](https://ibm-studios.slack.com/archives/C050J2L9M44/p1724687400779829?thread_ts=1724686786.420309&cid=C050J2L9M44), we can remove the full stops/periods from the captions mentioned in this PR.